src/client/components/sideMenubar/sideMenu.js

```diff
@@ -14,28 +15,33 @@ function SideMenu({ highLightItem }) {
             className="side_menu_whiteborder"
             data-highlighted={highLightItem === 0}
           >
-            <a href={highLightItem}>
-              <img src={homeImg} alt="homeImg" />
-            </a>
+            <Link to="/">
+              <a href={highLightItem}>
+                <img src={homeImg} alt="homeImg" />
+              </a>
+            </Link>
```

Comment:
You should not wrap the `<a>` in a `<Link>` as the will end up as two `<a>` which is not the correct HTML way.

See the example here: 
https://reactrouter.com/web/example/basic